examples/howto/server_embed/django_embed/manage.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'django_embed.settings')
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:
+        raise ImportError(
+            "Couldn't import Django. Are you sure it's installed and "
+            "available on your PYTHONPATH environment variable? Did you "
+            "forget to activate a virtual environment?"
```

Comment:
"conda or virtual" ?

Reply:
That was auto-generated by django. In any case, conda environment is just a implementation of virtual environment concept, so this language seems sufficient to me.